routes/web.php

```diff
@@ -35,14 +35,49 @@
     return view('pub.more');
 })->name('pub.more');
 
+
 Route::group(['prefix' => 'admin'], function(){
 
     Route::get('dashboard', function() {
         return view('admin.index');
     })->name('admin.index');
 
+	Route::get('categories', function() {
```

Comment:
absolute URls

Reply:
Relative URL doesn't make more sense, since we are in admin route group.

Reply:
okay, makes sense.